packages/nimble-components/src/mapping/base/index.ts

```diff
@@ -0,0 +1,14 @@
+import { attr } from '@microsoft/fast-element';
+import { FoundationElement } from '@microsoft/fast-foundation';
+import type { MappingKey } from './types';
+
+/**
+ * Base class for mapping configuration elements
+ */
+export abstract class Mapping extends FoundationElement {
+    @attr({ mode: 'fromView' })
+    public key?: MappingKey;
+
+    @attr()
+    public label?: string;
```

Comment:
Has there already been discussion on the proper name for this attribute, as `label` just seems like an overloaded term that seems a little odd here. Other options I can think of are `display`, `value`, or `text`. I do _not_ feel strongly about this, and `label` can certainly suffice. It just struck me as a slightly odd term.

Reply:
I don't think anyone brought this up before. I see your point, and I lean slightly towards changing it to `text`. I'll leave this comment open and urge other reviewers (please) to weigh in.

Reply:
I'm changing it to `text`.